src/main/java/ch/naviqore/service/impl/PublicTransitServiceImpl.java

```diff
@@ -36,8 +38,8 @@ public class PublicTransitServiceImpl implements PublicTransitService {
     private final KDTree<ch.naviqore.gtfs.schedule.model.Stop> spatialStopIndex;
     private final SearchIndex<ch.naviqore.gtfs.schedule.model.Stop> stopSearchIndex;
     private final WalkCalculator walkCalculator;
-    private final List<TransferGenerator.Transfer> additionalTransfers;
-    private final RaptorCache cache;
+    private final RaptorAlgorithm raptorAlgorithm;
+    private final GtfsTripMaskProvider tripMaskProvider;
 
     PublicTransitServiceImpl(ServiceConfig config, GtfsSchedule schedule,
```

Comment:
The `additionalTransfers` passed in the constructor are never used. But simply deleting the parameter seems not appropriate since the are still generated in the `PublicTransitServiceInitializer`:

```java
...
        if (config.getTransferTimeBetweenStopsMinimum() >= 0) {
            this.additionalTransfers = generateTransfers(schedule,
                    createTransferGenerators(config, walkCalculator, spatialStopIndex));
        } else {
            this.additionalTransfers = new ArrayList<>();
        }

...

    public PublicTransitService get() {
        return new PublicTransitServiceImpl(config, schedule, spatialStopIndex, stopSearchIndex, walkCalculator,
                additionalTransfers);
    }
```

Reply:
Fixed in a3e62da